frontend/src/layouts/authLayout.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { blue } from "@ant-design/colors";
+import { BoxPlotTwoTone } from "@ant-design/icons";
+import { Row, Col, Avatar } from "antd";
+
+import Illustration from "../pages/components/authIllustration";
+
+const AuthLayout = ({ children, illustration }) => {
```

Comment:
- exported component name is `AuthLayout` but filename is `authLayout.js`. Please capitalize the filename :man: